.travis.yml

```diff
@@ -1,20 +1,42 @@
 language: scala
 
 scala:
-  - 2.11.8
-  - 2.12.1
+- 2.11.8
```

Comment:
Does this whitespace change make the file parse differently? Not sure if it was intentional or not.